src/components/Sum.jsx

```diff
@@ -1 +1,23 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Sum({
+  a = 0,
+  b = 0,
+}) {
```

Comment:
```suggestion
export function Sum({ a, b }) {
```

if you have default props, you do not need to assign them here

Reply:
✔️